students/jvirtue/lesson02/grid.py

```diff
@@ -0,0 +1,57 @@
+# Print 2x2 grid for Part I
+""" horizontal = ("+"+"-"*4)*2
+vertical = ("|"+" "*4)*2
+
+print(horizontal,end="+")
+print()
+print(vertical,end="|")
+print()
+print(vertical,end="|")
+print()
+print(vertical,end="|")
+print()
+print(vertical,end="|")
+print()
+print(horizontal,end="+")
+print()
+print(vertical,end="|")
+print()
+print(vertical,end="|")
+print()
+print(vertical,end="|")
+print()
+print(vertical,end="|")
+print()
+print(horizontal,end="+") """
+
+#Print a grid based on parameter Part II
+#One parameter to control size of grid
+""" def printgrid(bars):
```

Comment:
Why the comment block?

For function names, they should be snake_case in Python per PEP8

Reply:
Made updates to this pull request #1347